src/utils/helpers.ts

```diff
@@ -0,0 +1,260 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { OP_PUSHDATA1 } from '../opcodes';
+import { BLOCK_VERSION, CREATE_TOKEN_TX_VERSION, DEFAULT_TX_VERSION, MERGED_MINED_BLOCK_VERSION, DECIMAL_PLACES } from '../constants';
+import buffer from 'buffer';
+import Long from 'long';
+import Transaction from '../models/transaction';
+import { crypto } from 'bitcore-lib';
+
+/**
+ * Helper methods
+ *
+ * @namespace Helpers
+ */
+
+const helpers = {
+  /**
+   * Get object type (Transaction or Block)
+   *
+   * @param {Transaction} tx Object to get the type
+   *
+   * @return {string} Type of the object
+   *
+   * @memberof Helpers
+   * @inner
+   */
+  getTxType(tx: Transaction): string {
+    if (this.isBlock(tx)) {
+      if (tx.version === BLOCK_VERSION) {
+        return 'Block';
+      } else if (tx.version === MERGED_MINED_BLOCK_VERSION) {
+        return 'Merged Mining Block';
```

Comment:
Spaced strings? Isn't it better to return an ENUM? Or put them in constants?

Reply:
Created an ENUM